recipe/meta.yaml

```diff
@@ -24,19 +25,21 @@ requirements:
     - setuptools
   run:
     - nltk
-    - numpy  # [not (linux and x86_64)]
-    - numpy >=1.22  # [linux and x86_64]
+    - numpy {{ numpy }}
```

Comment:
I would either keep the older values or use `numpy`. `{{numpy}}` should only be used in `host`.